tests/tck/features/ttl/TTL.feature

```diff
@@ -340,27 +350,27 @@ Feature: TTLTest
       | "player" | 'CREATE EDGE `player` (\n `id` int64 NULL,\n `name` string NULL,\n `age` int64 NULL,\n `address` string NULL,\n `score` float NULL\n) ttl_duration = 0, ttl_col = ""' |
     When executing query:
       """
-      ALTER EDGE player change(name int), drop(name);
+      ALTER EDGE player change(name string), drop(name);
```

Comment:
The error should come from the intended, not from illegal type change. So here I keep type the same.

Reply:
The same for the followings.